index.bs

```diff
@@ -45,6 +45,16 @@ application runs a machine learning model such as [[DeepLabv3+]] or
 [[MaskR-CNN]] to semantically split an image into segments and replaces
 segments that represent other people and background with another picture.
 
+A live streaming application offers [[Bullet-Chatting]], a user-generated dynamic
+comments float over a video or static image feature, which may disturb users from
+watching the media content. The [[Masking-Bullet-Chatting]] could semantically
+split video into masks by running a machine learning model such as [[DeepLabv3+]]
+or [[LDN]], and leverage CSS3 to avoid the live streaming anchors being overwritten
+by bullet chatting. Users will be able to see the expression, posture and movement
```

Comment:
s/Users will be able to see the expression, posture and movement
of anchors more clearly and conveniently.//

I think for these details we can defer to the proposal doc that is referenced.